utils/abilities.cfg

```diff
@@ -1962,81 +1995,81 @@ Enemy units cannot see this unit during dawn or dusk, except if they have units
     [/resistance]
 #enddef
 #define ABILITY_RECALL
-[teleport]
-    id=recall
-    name= _ "recall"
-    female_name= _ "female^recall"
-    description= _ "This unit may teleport to an leader of its side."
-    [tunnel]
-        bidirectional=false
-        [source]
-            x=$teleport_unit.x,y=$teleport_unit.y
-        [/source]
-        [target]
+    [teleport]
+        id=recall
+        name= _ "recall"
+        female_name= _ "female^recall"
+        description= _ "This unit may teleport to an leader of its side."
+        [tunnel]
+            bidirectional=false
+            [source]
+                x=$teleport_unit.x,y=$teleport_unit.y
+            [/source]
```

Comment:
Did I commit some bad indentation here?

Reply:
Could have been me, I've been doing a lot of cut and paste merging.